backend/schemas/sources/Base.libsonnet

```diff
@@ -102,6 +105,77 @@ local Enum = {
     ],
     title: 'EmptyString_Null',
   },
+  UnitedStatesStateAbbr: Types.string {
```

Comment:
This one we should probably ticket as a bug, or fix now... does this include all of the territories? I don't see PR, for example.

Reply:
It does not, and I currently can't recall whether that's because it's how Census did it or because at some point we hurried through adding this list.